<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/data/images/find.png</normaloff>:/data/images/find.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="locale">
   <locale language="Arabic" country="SaudiArabia"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>فهرس</string>
    </property>
    <widget class="QMenu" name="menuIndexesList">
     <property name="title">
      <string>اختيار فهرس</string>
     </property>
    </widget>
    <addaction name="actionNewIndex"/>
    <addaction name="actionEditIndexes"/>
    <addaction name="menuIndexesList"/>
    <addaction name="actionIndexInfo"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>اعدادات</string>
    </property>
    <addaction name="actionSearchSettings"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>مساعدة</string>
    </property>
    <addaction name="actionLogDialog"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>البحث</string>
    </property>
    <widget class="QMenu" name="menuSearchFields">
     <property name="title">
      <string>مجال البحث</string>
     </property>
     <addaction name="actionSaveSelectedField"/>
     <addaction name="actionEditField"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="actionSelectAllBooks"/>
    <addaction name="actionUnSelectallBooks"/>
    <addaction name="separator"/>
    <addaction name="actionSelectVisibleBooks"/>
    <addaction name="actionUnselectVisisbleBooks"/>
    <addaction name="separator"/>
    <addaction name="actionCollapseFilter"/>
    <addaction name="actionExpandFilter"/>
    <addaction name="separator"/>
    <addaction name="menuSearchFields"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionNewIndex">
   <property name="text">
    <string>انشاء فهرس...</string>
   </property>
  </action>
  <action name="actionIndexInfo">
   <property name="text">
    <string>معلومات حول الفهرس</string>
   </property>
  </action>
  <action name="actionSearchSettings">
   <property name="text">
    <string>اعدادات...</string>
   </property>
  </action>
  <action name="actionEditIndexes">
   <property name="text">
    <string>تعديل الفهارس...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>حول البر نامج</string>
   </property>
  </action>
  <action name="actionLogDialog">
   <property name="text">
    <string>نافذة الاخطاء</string>
   </property>
  </action>
  <action name="actionSelectAllBooks">
   <property name="text">
    <string>اختيار كل الكتب</string>
   </property>
  </action>
  <action name="actionUnSelectallBooks">
   <property name="text">
    <string>الغاء كل الكتب</string>
   </property>
  </action>
  <action name="actionSelectVisibleBooks">
   <property name="text">
    <string>اختيار الكتب الظاهرة</string>
   </property>
  </action>
  <action name="actionUnselectVisisbleBooks">
   <property name="text">
    <string>الغاء الكتب الظاهرة</string>
   </property>
  </action>
  <action name="actionCollapseFilter">
   <property name="text">
    <string>ضغط الشجرة</string>
   </property>
  </action>
  <action name="actionExpandFilter">
   <property name="text">
    <string>عرض الشجرة</string>
   </property>
  </action>
  <action name="actionSaveSelectedField">
   <property name="text">
    <string>حفظ الكتب المحدد...</string>
   </property>
  </action>
  <action name="actionEditField">
   <property name="text">
    <string>تعديل مجالات البحث...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
